elrond-wasm/src/storage/mappers/vec_mapper.rs

```diff
@@ -181,6 +181,31 @@ where
         storage_clear(self.api.clone(), &self.item_key(index));
     }
 
+    /// Clears item at index from storage by swap remove
+    /// last item takes the index of the item to remove
+    /// and we remove the last index.
+    pub fn swap_remove(&mut self, index: usize) -> bool {
+        let _ = self.swap_remove_and_get_old_last(index);
+        true
+    }
```

Comment:
I don't think we need to return `true` here?
```suggestion
    pub fn swap_remove(&mut self, index: usize) {
        let _ = self.swap_remove_and_get_old_last(index);
    }
```